ci/run-k8s-e2e-tests.sh

```diff
@@ -31,8 +31,8 @@ RUN_NETWORK_POLICY=false
 RUN_E2E_FOCUS=""
 RUN_E2E_SKIP=""
 KUBECONFIG_OPTION=""
-DEFAULT_E2E_CONFORMANCE_FOCUS="\[Conformance\]"
-DEFAULT_E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]"
+DEFAULT_E2E_CONFORMANCE_FOCUS="\[sig-network\]|\[Conformance\]"
+DEFAULT_E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[GCE\]|\[Ingress\]|\[NetworkPolicy\]|\[IPv6\]|\[Feature:.+\]|\[Feature:IPv6DualStack\]|\[Feature:IPv6DualStackAlphaFeature\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]|should create pod that uses dns|should provide Internet connection for containers"
```

Comment:
This script is used by CAPV cluster, GKE, EKS, AKS. If the focus and skip list (especially the latter) are specific to some testbeds, should they be updated in the script that runs the test in that kind of testbeds?

Reply:
Thanks for the reminder. We only introduce signetwork test for capv cluster in this PR. I added a new parameter to separate signetwork test from conformance test. @tnqn